jbpm-wb-case-mgmt/jbpm-wb-case-mgmt-client/src/main/java/org/jbpm/workbench/cm/client/list/CaseInstanceViewImpl.html

```diff
@@ -1,6 +1,6 @@
 <div class="list-group-item" data-field="list-item">
     <div class="list-view-pf-actions">
-        <button class="btn btn-default hidden" data-field="complete" data-i18n-key="Complete"></button>
+        <button class="btn btn-primary hidden" data-field="cancel" data-i18n-key="Cancel"></button>
```

Comment:
@bxf12315 I don't think we want to change the button style to primary in here, start case is the main action in the case list screen.

Reply:
@cristianonicolai  If I change to 'cancel', the button is not blue button.
 I want to the blue button ,so I change the style.

Reply:
@bxf12315 a "blue"button is often used as the primary action in a page, to give the user its main focus. According to the designs we had for this screen, the primary button is located at the top of the list with the "start case" action, if you change this cancel button to be a primary ( blue ) button we would break that user experience.

Reply:
ok go it